barretenberg/cpp/CMakeLists.txt

```diff
@@ -16,14 +16,11 @@ configure_file(
     @ONLY
 )
 
-
 option(DISABLE_ASM "Disable custom assembly" OFF)
 option(DISABLE_ADX "Disable ADX assembly variant" OFF)
 option(MULTITHREADING "Enable multi-threading" ON)
 option(OMP_MULTITHREADING "Enable OMP multi-threading" OFF)
-option(TESTING "Build tests" ON)
```

Comment:
I have removed the TESTING and BENCHMARK options in favour of simplifying the build pipeline, but I'm happy to revert it if desired